scripts/build-host

```diff
@@ -41,6 +41,7 @@ protobuf_CONFIGURE	:= --prefix=$(PREFIX) --disable-shared
 libsodium_CONFIGURE	:= --prefix=$(PREFIX) --disable-shared
 opus_CONFIGURE		:= --prefix=$(PREFIX) --disable-shared
 libvpx_CONFIGURE	:= --prefix=$(PREFIX) --disable-examples --disable-unit-tests --enable-pic
+msgpack_CONFIGURE	:= -DCMAKE_INSTALL_PREFIX:PATH=$(PREFIX)
```

Comment:
I don't know how critical it is, but should the `-DBUILD_SHARED_LIBS=OFF` argument also be here?

Reply:
Doesn't make a difference since it's only on Android we only upload specific .so-files, but updated to match the Android configuration. :P Looks like that's how I had it set up before anyway.